CONTRIBUTING.md

```diff
@@ -1,7 +1,10 @@
 <p align="center">
-  <a href="https://sentry.io" target="_blank" align="center">
+  <a href="https://sentry.io#gh-light-mode-only" target="_blank" align="center">
     <img src="https://sentry-brand.storage.googleapis.com/sentry-logo-black.png" width="280">
```

Comment:
In the Unity repo we went with an SVG:

https://github.com/getsentry/sentry-unity/blob/3b7299b17cc3f231aae3f6f7b6c6128662c6e6b8/README.md?plain=1#L2-L4